openpgp/packet/public_key.go

```diff
@@ -598,7 +598,7 @@ func (pk *PublicKey) CanSign() bool {
 	return pk.PubKeyAlgo != PubKeyAlgoRSAEncryptOnly && pk.PubKeyAlgo != PubKeyAlgoElGamal
 }
 
-// VerifySignature returns nil iff sig is a valid signature, made by this
+// VerifySignature returns nil if sig is a valid signature, made by this
```

Comment:
I had no idea. I'll change these back. I run my editor with spell-checker and I occasionally correct a comment if it's near code I'm working on.